.github/workflows/ci-linux.yml

```diff
@@ -21,17 +21,17 @@ jobs:
       - name: Install dependencies
         run: |
           sudo apt-get update
-          sudo apt-get install -y libsdl2-dev libsdl2-ttf-dev libasound2-dev libxinerama-dev libxi-dev qt5-default
+          sudo apt-get install -y libsdl2-dev libsdl2-ttf-dev libasound2-dev libxinerama-dev libxi-dev qt5-default cmake
       - name: Install clang
         if: matrix.compiler == 'clang'
         run: sudo apt-get install -y clang
       - name: Build
         env:
-          OVERRIDE_CC: ${{ matrix.cc }}
-          OVERRIDE_CXX: ${{ matrix.cxx }}
-          ARCHOPTS: ${{ matrix.archopts }}
-          TOOLS: 1
-        run: make -j2
+          CFLAGS: ${{ matrix.archopts }}
+          CXXFLAGS: ${{ matrix.archopts }}
+        run: |
+          CC=${{ matrix.cc }} CXX=${{ matrix.cxx }} cmake . -DTOOLS=ON
```

Comment:
You should set environment variables in the "env" section.

Reply:
Thanks, aware of that, but was wondering if better to be set as -DCMAKE_C_COMPILER  and -DCMAKE_CXX_COMPILER or doing it like thing, not a problem to change this, just to make sure first it builds fine on CI